lms/djangoapps/instructor_task/models.py

```diff
@@ -212,7 +212,8 @@ def from_config(cls, config_name):
             return DjangoStorageReportStore(
                 storage_class='django.core.files.storage.FileSystemStorage',
                 storage_kwargs={
-                    'location': config['ROOT_PATH'],
+                    'location': config['ROOT_PATH'] or getattr(settings, 'EDX_GRADES_STORAGE_MEDIA_ROOT', None),
```

Comment:
You still doing the same job twice, please create a variable for `getattr(settings, 'EDX_GRADES_STORAGE_MEDIA_ROOT', None)` and use it in the required places.

Thanks.